Community/Tdarr_Plugin_Greg_MP3_FFMPEG_CPU.js

```diff
@@ -0,0 +1,44 @@
+const importFresh = require('import-fresh');
+
+module.exports.details = function details() {
+  return {
+    id: 'Tdarr_Plugin_Greg_MP3_FFMPEG_CPU',
+    Description: '[Contains built-in filter] Convert an audio file to mp3, retaining ID3 tags, '
+    + 'and at original bitrate up to 320k - from type of: "flac,wav,ape,ogg,m4a,wma,opus" ',
+    Link: 'https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Community/Tdarr_Plugin_Greg_MP3_FFMPEG_CPU.js',
+    Name: 'Audio Transcode to MP3 using CPU and FFMPEG',
+    Operation: 'Transcode',
+    Tags: 'pre-processing,ffmpeg,audio only,mp3',
```

Comment:
I missed this, but mp3 isn't a valid tag. Valid ones are listed in the example plug-in https://github.com/HaveAGitGat/Tdarr_Plugins/blob/master/Tdarr_Plugin_aaaa_Pre_Proc_Example.js